ThemingCodelab/app/src/main/java/com/codelab/theming/ui/start/Home.kt

```diff
@@ -174,6 +186,7 @@ fun PostItem(
         icon = {
             Image(
                 bitmap = imageResource(post.imageThumbId),
+                contentDescription = stringResource(R.string.label_post_image)
```

Comment:
This image is currently used only for decorative purposes and does not represent a meaningful action that a user can take. Therefore you want to set its contentDescription to `null`.